.gitignore

```diff
@@ -3,3 +3,4 @@
 /dist
 /node_modules
 /packages/*/dist
+.idea
```

Comment:
(for consistency: `/.idea`)

Reply:
I have amended the commit and corrected as suggested